src/lib/logical_query_plan/sort_node.cpp

```diff
@@ -31,6 +31,14 @@ std::string SortNode::description() const {
   return stream.str();
 }
 
+size_t SortNode::_on_hash() const {
+  size_t hash{0};
+  for (const auto& order_by_mode : order_by_modes) {
+    boost::hash_combine(hash, boost::hash_value(order_by_mode));
```

Comment:
Did we need hash_value? If not, please check the diff if was left in somewhere else, too